src/screens/Inbox.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { View, Text } from 'react-native';
 
-export default class Inbox extends React.Component {
+import MessageListContainer from '../components/message/MessageListContainer';
+
+class Inbox extends React.Component {
   render() {
     return (
-      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-        <Text>Inbox Screen</Text>
-      </View>
+      <MessageListContainer />
```

Comment:
this isn't going to look like this when we add tabs (messages + notifications) so let's revert this to the placeholder text for now. it's cool to do this for testing purposes but what we end up pushing should ideally only move us towards the final product vision